src/AssetManager.php

```diff
@@ -515,10 +515,6 @@ private function registerAssetBundle(string $name, int $position = null): void
      */
     private function registerFiles(string $bundleName): void
     {
-        if (!isset($this->registeredBundles[$bundleName])) {
```

Comment:
What if there's no such bundle when this method is called?

Reply:
Bundle always registred, because in all places where used this private method before him run `registerAssetBundle`.